editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-element-renderer-utils.tsx

```diff
@@ -78,13 +79,13 @@ export function renderCoreElement(
       if (originalIDForProps != null) {
         passthroughProps[UTOPIA_ORIGINAL_ID_KEY] = originalIDForProps
       }
-      const originalParentIDForProps = Utils.defaultIfNull(
-        assembledProps[UTOPIA_UID_ORIGINAL_PARENTS_KEY],
-        parentComponentInputProps[UTOPIA_UID_ORIGINAL_PARENTS_KEY],
-      )
-      if (originalParentIDForProps != null) {
-        originalParentIDForProps[UTOPIA_UID_ORIGINAL_PARENTS_KEY] = originalParentIDForProps
-      }
+      // const originalParentIDForProps = Utils.defaultIfNull(
```

Comment:
👀

Reply:
so I noticed that this is not used, but I wanted to ask if anyone knows if this is an accident or not. @Rheeseyb  or @seanparsons  maybe you remember something about this?

Reply:
It's still used here right? https://github.com/concrete-utopia/utopia/blob/master/editor/src/components/canvas/dom-walker.ts#L323

Reply:
ouch! so `const originalParentIDForProps` was not read anywhere in this function. I guess this means that this probably regressed sometime in the past and we didn't notice it?

Reply:
Or it was written wrong in the first place.